### ISSUE TITLE
error when using Postgresql

### ISSUE THREAD

**Issue by billspat** [krishnanlab/geneplexus_app#243]
so far we've tested with Sqlite backend database.   

When attempting to run using Postgresql with the current models (after much trial getting python package psycopg2 to install on an M1 mac in Rosetta mode), I get this error: 

```
psycopg2.errors.DatatypeMismatch: foreign key constraint "results_jobname_fkey" cannot be implemented
DETAIL:  Key columns "jobname" and "jobid" are of incompatible types: integer and character varying.
```

SQLite  is notoriously un-picky about column types, but PG is extra picky.   Solution is to review the models and see if we can make the data types consistent in a way the works with both SQLite and PG.

**Comment by billspat** [comment]
there is another error related to postgresql that I'm adding to this issue since it affects the same module, but it's cause by logging in with github.  

When logging in to Github, there is an error that the password hash field is too short.  Currently in the models the string fields are limited to 64 chars.  to fix I suggest removing the length on that field (both PG and sqlite allow for null length for string types which means unlimited length).

**Comment by billspat** [comment]
this was fixed by PRs #249 and #255
